specs/engage-digital_postman2.json

```diff
@@ -170,9 +170,9 @@
                       "disabled": true
                     },
                     {
-                      "key": "label",
+                      "key": "name",
```

Comment:
I think we're supposed to manually change [`specs/engage-digital_openapi3.yaml`](https://github.com/ringcentral/engage-digital-api-docs/blob/master/specs/engage-digital_openapi3.yaml#L6234) and then generate `specs/engage-digital_postman2.json` from it (cf. [README](https://github.com/ringcentral/engage-digital-api-docs/tree/master/specs#engage-digital-api-specifications)).